runtime/onert/core/include/backend/ITensorRegistry.h

```diff
@@ -110,12 +110,9 @@ template <typename T_Tensor> class PortableTensorRegistryTemplate : public ITens
 
   bool setMigrantTensor(const ir::OperandIndex &ind, const std::shared_ptr<IPortableTensor> &tensor)
   {
-    // TODO Uncomment this as two tensors for an index is not allowed.
-    //      But now it is temporarily allowed as a workaround. External one hides Managed one.
-    // auto itr = _native.find(ind);
-    // if (itr != _native.end() && itr->second != nullptr && tensor != nullptr)
-    //  throw std::runtime_error{
-    //      "Tried to set an migrant tensor but an native tensor already exists."};
+    auto itr = _native.find(ind);
+    if (itr != _native.end() && itr->second != nullptr && tensor != nullptr)
```

Comment:
`tensor != nullptr` may need another error message? 
(or maybe simply `assert(tensor)`)

Reply:
I just remembered what that means. If tensor is null, I thought it means unsetting the tensor. I'm not sure that's necessary, though.

Reply:
If so, this condition should be changed to like below
```suggestion
    if (itr != _native.end() || tensor == nullptr)
```

Reply:
After offline discussion with @ragmani , we concluded that null entry in TensorRegistry is not allowed. Then the condition will be way simpler. Thank you for pointing that out!